Update: Trino, Presto Logic and Doc (#9859)
open-metadata/OpenMetadata: openmetadata-docs/content/connectors/database/trino/index.md

To run the Ingestion via the UI you'll need to use the OpenMetadata Ingestion Container, which comes shipped with
custom Airflow plugins to handle the workflow deployment.

<Note>

> To Inesget metadata from the Trino User Must have select privileges to this tables.

Reviewer: ```suggestion
To ingest metadata from the Trino source, the user must have select privileges for the following tables.
```
